content/800-data-platform/010-about/index.mdx

```diff
@@ -48,11 +48,11 @@ Learn the details about the Data Proxy in our [documentation](/data-platform/dat
 
 ### Feedback and Support
 
-You [can reach us via Intercom](/data-platform/contact-support) on the Prisma Data Platform application.
+You [can reach us via Zendesk](/data-platform/contact-support) on the Prisma Data Platform application.
```

Comment:
Suggestion to not mention the support platform, but just the way to reach it:
```suggestion
You [can reach us via "Support"](/data-platform/contact-support) on the Prisma Data Platform application.
```

Reply:
Ah, just noticed the next sentence - that would indeed not sound great to mention "Support" twice. 

Maybe someone can come up with a nicer way to say this anyway - if this runs via Intercom or Zendesk doesn't matter much I think.

Reply:
I've rephrased those now so there is no duplication.